docs/training-models.rst

```diff
@@ -105,9 +115,26 @@ Or an xgboost model::
             "name": "my_xgboost_model",
             "model": {
                 "type": "model/xgboost+json",
-                "definition": "[  { "nodeid": 0, "depth": 0, "split": "tmdb_multi", "split_condition": 11.2009,                     "yes": 1, "no": 2, "missing": 1, "children": [
-                                    { "nodeid": 1, "depth": 1, "split": "tmdb_title", "split_condition": 2.20631, "yes": 3, "no": 4, "missing": 3, "children": [
-                                    { "nodeid": 3, "leaf": -0.03125 },
+                "definition": "[  { \"nodeid\": 0, \"depth\": 0, \"split\": \"tmdb_multi\", \"split_condition\": 11.2009, \"yes\": 1, \"no\": 2, \"missing\": 1, \"children\": [
+                                    { \"nodeid\": 1, \"depth\": 1, \"split\": \"tmdb_title\", \"split_condition\": 2.20631, \"yes\": 3, \"no\": 4, \"missing\": 3, \"children\": [
+                                      { \"nodeid\": 3, \"leaf\": -0.03125 },
+                                    ..."
+            }
+        }
+    }
+
+Or an xgboost model with parameters::
+
+    POST _ltr/_featureset/more_movie_features/_createmodel
+    {
+        "model": {
+            "name": "my_xgboost_model",
+            "model": {
+                "type": "model/xgboost+json",
+                "definition": "{ \"objective\": \"reg:logistic\" }
```

Comment:
this does not seem to be a valid JSON.
Since XGBoost does not seem to define a clear json schema for its model and the only doc I could find is the list of split definitions directly I suggest to start defining one.

The XGBoost parser would have to be able to parse:
- existing format that starts directly with an array of split defitions
- custom format that allows passing params:
```
{
   "objective": "...",
   "splits": [
   ]
}
```